packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/UpdateDetailModalHandler.tsx

```diff
@@ -33,7 +33,12 @@ export const UpdateDetailModalHandler = ({
   const { addSuccess } = useMessageContext();
   const { id } = useParams();
   const { data: vcdOrganization } = useVcdOrganization({ id });
-  const { updateDetails, error, isError } = useUpdateVcdOrganizationDetails({
+  const {
+    updateDetails,
+    error,
+    isError,
+    isPending,
```

Comment:
no need to check isLoading as well ?

Reply:
`isLoading` is not present here as it's a mutation, but `isPending` is the equivalent props to `isLoading` of a query